ubuntu_init_dev_setup.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/bash
+
+# from https://tutorials.technology/solved_errors/10-fatal-error-libxml-xpath_h-No-such-file-or-directory.html
+sudo apt-get install libxml2-dev libxslt1-dev python-dev
```

Comment:
This shouldn't be in an separate file. Were you getting this error when running `make init`?

Reply:
It has been a while, but: I believe so, yes.